Assets/Scripts/Background/BeyondBorderBackgroundResizer.cs

```diff
@@ -0,0 +1,83 @@
+// Thanks to Loek van den Ouweland for scaling code explanation;
+// their article is where I came across a script with similar code for obtaining camera values and scaling a background.
+// van den Ouweland, L. (2019, October 9). STRETCH A UNITY SPRITE TO FILL THE SCREEN IN A 2D GAME (GAMEOBJECT, NOT UI-CANVAS). 
+// Loek van den Ouweland Software Engineering. Retrieved March 20, 2023, from 
+// https://www.loekvandenouweland.com/content/stretch-unity-sprite-to-fill-the-screen.html.

```

Comment:
I think the code is dissimilar enough that you could remove this citation, but can keep it if you want.

Reply:
I decided to shorten it to just a "thanks" and a web link, since the long citation did feel unnecessary when the code is very dissimilar at this point.